### PR TITLE
Document keyring support for 1.15

### PR DESCRIPTION
**Pull request by Joe-Winchester**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [x] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [x] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

**fix** #1392 

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/user-guide/configure-certificates-keyring.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+After running `ZWEKRING` successfully, the script `zowe-setup-certificates.sh` needs to be run which will create the `KEYSTORE_DIRECTORY` in USS.  Depending on how the `ZWEKRING` member has been customized the configuration file `zowe-setup-certificates.env` that is used to create the `KEYSTORE_DIRECTORY` will need to be customized.
+
+### ZWEKRING
+
+The `ZWEKRING` JCL helps you create a keyring containing a certificate and a local certificate authority that is used to self-sign the certificate.  Other scenarios (such as importing an existing certificate) are described in the `ZWEKRING` JCL with the commands commented out) but are currently unsupported and included in beta mode for early testing and feedback with Zowe 1.15.  If you find any issues please raise an issue at https://github.com/zowe/zowe-install-packaging/issues.  Future releases of Zowe will provide support for more keyring scenarios.
```
Comment (stevenhorsman) [inline docs/user-guide/configure-certificates-keyring.md:9]: This implies to me that scenario 3 is supported and not beta

File: docs/user-guide/configure-certificates-keyring.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+The `ZWEKRING` JCL helps you create a keyring containing a certificate and a local certificate authority that is used to self-sign the certificate.  Other scenarios (such as importing an existing certificate) are described in the `ZWEKRING` JCL with the commands commented out) but are currently unsupported and included in beta mode for early testing and feedback with Zowe 1.15.  If you find any issues please raise an issue at https://github.com/zowe/zowe-install-packaging/issues.  Future releases of Zowe will provide support for more keyring scenarios.  
+
+The `PRODUCT` variable specifies the z/OS security manager.  The default is `RACF` and should be changed to `ACF2` or `TSS` if you are using Access Control Facility CA-ACF2 or CA Top Secret for z/OS as your z/OS security manager.  
+
+```
+//         SET  PRODUCT=RACF         * RACF, ACF2, or TSS
+```
+
+If you are using a security manager other than RACF you will need to comment out the RACF security commands and uncomment the commands for your security manager.  This is described more in the JCL member `ZWEKRING`
```
Comment (stevenhorsman) [inline docs/user-guide/configure-certificates-keyring.md:17]: I don't think this is the case and no commenting is required - I believe you just change PRODUCT=ACF2 and it runs the correct DD @vit-tomica - can you confirm?

Comment (vit-tomica) [reply docs/user-guide/configure-certificates-keyring.md]: That's correct. No need to comment out anything here.

File: docs/user-guide/configure-certificates-keyring.md (hunk 1 above)
Comment (Joe-Winchester) [reply docs/user-guide/configure-certificates-keyring.md]: I've added a para to configure-certificates.md to call out that keyrings are beta tech preview,  add the same para to the beginning of configure-certificate-keyring.md, and also change the chapter title to include (Beta Technical Preview) in bold.

File: docs/user-guide/configure-certificates-keyring.md (hunk 2 above)
Comment (Joe-Winchester) [reply docs/user-guide/configure-certificates-keyring.md]: Good catch - thank you for the correction.  I wasn't aware that just the PRODUCT= change alone was able to run the correct DDs by itself.   Removed the offending para.

**Pull request merged**